drm/ResourceManager.cpp

```diff
@@ -71,6 +72,40 @@ void ResourceManager::Init() {
       if (stat(path.str().c_str(), &buf) != 0)
         break;
 
+      node_num++;
+    }
+
+    // only have one card, is BM/GVT-d/Virtio
+    if (node_num == 1) {
+      std::ostringstream path;
+      path << path_pattern << 0;
+      auto dev = DrmDevice::CreateInstance(path.str(), this);
+      if (dev) {
+        drms_.emplace_back(std::move(dev));
+      }
+    }
+    // is SR-IOV or iGPU + dGPU
```

Comment:
add more comments, like:
For SR-IOV case, use virtio-gpu(card1) for display
For iGPU+dGPU case, only use iGPU(card0) for display